app/models/facility.rb

```diff
@@ -108,7 +104,7 @@ def facilities
   end
 
   def cohort_analytics(period, prev_periods)
-    query = CohortAnalyticsQuery.new(registered_hypertension_patients)
+    query = CohortAnalyticsQuery.new(assigned_hypertension_patients)
```

Comment:
Apart from naming, I believe this is the meaty change of this PR.

Reply:
This is one meaty change, the other one is grouping by `:assigned_facility_id` in `patient_counts`.

Reply:
Ah, yes. 👍